google/data_source_google_service_account_key_test.go

```diff
@@ -50,7 +94,39 @@ resource "google_service_account_key" "acceptance" {
 	public_key_type = "TYPE_X509_PEM_FILE"
 }
 
+data "google_service_account_key" "acceptance" {
+	name = "${google_service_account_key.acceptance.id}"
```

Comment:
I really really want to get us away from the practice of relying on the way an id is shaped, since it limits our availability to change them in the future. I just filed #1927 yesterday about that, but in the meantime, is there something else that would work here?